README.md

```diff
@@ -11,6 +11,8 @@ To clone and run this repository you'll need [Git](https://git-scm.com) and [Nod
 ```bash
 # Clone this repository
 git clone https://github.com/nodegui/react-nodegui-starter
+# Install CMake
+brew install cmake
```

Comment:
can you add a comment also saying this is for mac or maybe add installation step for window and linux too